compiler-core/src/language_server/tests/mod.rs

```diff
@@ -82,6 +82,21 @@ impl LanguageServerTestIO {
         path
     }
 
+    pub fn hex_dep_module(&self, dep: &str, name: &str, code: &str) -> Utf8PathBuf {
+        let path = self
+            .paths
+            .root()
+            .join("build")
+            .join("packages")
+            .join(dep)
+            .join("src")
+            .join(name)
+            .with_extension("gleam");
```

Comment:
Please use the method on the path class rather than duplicating this logic here.